README.md

```diff
@@ -2,10 +2,10 @@
 
 [![Documentation Status](https://readthedocs.org/projects/docs-community/badge/?version=latest)](https://docs-community.readthedocs.io/en/latest/?badge=latest)
 
-The Python Documentation Working Group (Docs WG) was created and accepted by the **Python Steering Council** in 2021
-to help bootstrap a larger documentation team and community.
+The Python Documentation Editorial Board (Docs EB) was created and accepted by the **Python Steering Council** in 2023
+in [PEP 732](https://peps.python.org/pep-0732/) to help bootstrap a larger documentation team and community.
 
-This repo serves as documentation for the Docs WG, which is funny, in a recursive kind of way.
+This repo serves as documentation for the Docs EB, which is funny, in a recursive kind of way.
```

Comment:
Hm. The EB has its own repo (https://github.com/python/editorial-board) already, so we should probably say something here about how we expect to use the two separate repos.

@willingc may be able to say it better than me; my intuition tells me that it's something about governance vs. execution but I would need help turning that into a useful parapgraph.

(Also, I recommend removing the "which is funny, ..." part-- it's not *that* funny, and people who are tickled by that sort of stuff don't need to be told that it is. :-)

Reply:
Also isn't the EB supposed to be [a subset](https://peps.python.org/_images/pep-0732-concentric.drawio.svg) -- not a replacement -- of the WG?

My understanding is that the WG still exists unchanged and this is the repo for the WG, and now there's also the EB (which is part of the WG) with its own separate repo.

(Also the fact that you are suggesting to remove "which is funny, ..." is funny, in a recursive kind of way :upside_down_face: )

Reply:
@hugovk Maybe something like this may clear this up:

- Add the image referenced by @ezio-melotti 
- Users of Python Documentation
- Documentation Community Group (Working Group created in 2021 by the Python Steering Council)
   - Referred to as the Documentation Community or docs-community (to minimize confusion we will stop referring to this as a Working Group
   - Repo (this repo): https://github.com/python/docs-community
- Editorial Board (Approved in 2023 by Steering Council and described in PEP 732)
   - Referred to as Editorial Board or EB
   - Repo: https://github.com/python/editorial-board

Reply:
Yeah and I would remove the "which is funny". Although it is Ezio, I think we should lean into communicating to folks that will not get the reference :wink:

Reply:
Something like this?

https://github.com/python/docs-community/pull/99/commits/08829ea88be291b9d99a5f1bea6a1747cec5cc95

We'll have to do something about the SVG's transparent background, because it doesn't show up in dark mode: https://github.com/python/docs-community/blob/08829ea88be291b9d99a5f1bea6a1747cec5cc95/README.md


<img width="636" alt="image" src="https://github.com/python/docs-community/assets/1324225/1c5441db-981f-4b0d-bb08-e64512adcb77">


The PEPs site adds a CSS class to invert it, we can't do that in a README. Maybe just set the background to solid white here?